examples/pytorch/vision/Conference_Room_M4/README.md

```diff
@@ -0,0 +1,129 @@
+# Code for Face Detection experiments with RNNPool
```

Comment:
This README seems to be an extension of Face_Detection/README.md with the addition of downloading SCUTHEAD and finetuning and testing with it, Can we consolidate this folder to Face_Detection with extra steps/options for _M4 model?